test/index_test.js

```diff
@@ -28,10 +28,10 @@ describe("KintoAdmin", () => {
       buckets: [{}],
       serverInfo: {},
     };
-    const createKintoAdmin = () =>
+    const createKintoAdmin = (settings = {}) =>
       createComponent(KintoAdmin, {
         plugins: [],
-        settings: {},
+        settings: settings,
```

Comment:
nit: with ES6 `{settings}` is enough

Reply:
you mean just `settings` right? Like so:

```javascript
    const createKintoAdmin = (settings = {}) =>
      createComponent(KintoAdmin, {
        plugins: [],
        settings,
      });
```

Reply:
yes